docs/HowTo/Deploy/Bootnodes.md

```diff
@@ -0,0 +1,41 @@
+description: Configuring bootnodes in production networks    
+<!--- END of page meta data -->
+
+# Configuring Bootnodes in a Production Network 
+
+A network must have at least one operating bootnode. To allow for continuity in the event of failure, 
+configure more than one bootnode. 
+
+We do not recommend putting bootnodes behind a load balancer. We recommend putting more bootnodes on the network itself. 
+
+The [enode](../../Concepts/Node-Keys.md#enode-url) of a bootnode is tied to the node public key and IP address. 
+To simplify recovering from complete bootnode failure: 
+
+1. Create the [node key pair](../../Concepts/Node-Keys.md) (that is, the private and public key) before starting the bootnode and put
+the private key in the `/opt/pantheon` directory.
```

Comment:
@joshuafernandes - is this assuming you're deploying in a specific manner?  ie, the /opt/pantheon directory is not the default location for the private key when you install locally.